scalastyle-config.xml

```diff
@@ -1,7 +1,7 @@
 <scalastyle>
     <name>Scalastyle standard configuration</name>
     <check level="error" class="org.scalastyle.file.FileTabChecker" enabled="true"></check>
-    <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="true">
+    <check level="error" class="org.scalastyle.file.FileLengthChecker" enabled="false">
```

Comment:
Does this need to be part of this PR? Which file is too long all of a sudden?

Reply:
NodeBuilder.scala, but the limit is 800 so I wouldn't call that "all of a sudden." IMHO that is close enough to disabling this rule, so that's why this is the next logical step. ;)

Reply:
Fair enough :)

Reply:
I see people do this to suppress this in one file they don't want to deal with:`
```scala
// scalastyle:off number.of.methods method.length file.size.limit
trait Blockchain
```